Prog/BPK Forecasts_2010-45.sas

```diff
@@ -0,0 +1,98 @@
+				for Census tracts within the Bridge Park geography. 
+
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( Planning )
+%DCData_lib( Bridgepk )
+%DCData_lib( ACS )
+%DCData_lib( General )
+
+
+data bpk_popforecasts;
+	set planning.Pop_forecast_r9_dc_tr10;
+	/*Percent change in employment for base year 2010*/
+		e10_15pct = ((emp2015 - emp2010)/emp2010)*100;
+		e10_20pct = ((emp2020 - emp2010)/emp2010)*100;
+		e10_25pct = ((emp2025 - emp2010)/emp2010)*100;
+		e10_30pct = ((emp2030 - emp2010)/emp2010)*100;
+		e10_35pct = ((emp2035 - emp2010)/emp2010)*100;
+		e10_40pct = ((emp2040 - emp2010)/emp2010)*100;
+		e10_45pct = ((emp2045 - emp2010)/emp2010)*100;
+
+	/*Percent change in employment for base year 2015*/
+		e15_20pct = ((emp2020 - emp2015)/emp2015)*100;
+		e15_25pct = ((emp2025 - emp2015)/emp2015)*100;
+		e15_30pct = ((emp2030 - emp2015)/emp2015)*100;
+		e15_35pct = ((emp2035 - emp2015)/emp2015)*100;
+		e15_40pct = ((emp2040 - emp2015)/emp2015)*100;
+		e15_45pct = ((emp2045 - emp2015)/emp2015)*100;
+
+	/*Percent change in households for base year 2010*/
+		hh10_15pct = ((hh2015 - hh2010)/hh2010)*100;
+		hh10_20pct = ((hh2020 - hh2010)/hh2010)*100;
+		hh10_25pct = ((hh2025 - hh2010)/hh2010)*100;
+		hh10_30pct = ((hh2030 - hh2010)/hh2010)*100;
+		hh10_35pct = ((hh2035 - hh2010)/hh2010)*100;
+		hh10_40pct = ((hh2040 - hh2010)/hh2010)*100;
+		hh10_45pct = ((hh2045 - hh2010)/hh2010)*100;
+
+	/*Percent change in household for base year 2015*/
+		hh15_20pct = ((hh2020 - hh2015)/hh2015)*100;
+		hh15_25pct = ((hh2025 - hh2015)/hh2015)*100;
+		hh15_30pct = ((hh2030 - hh2015)/hh2015)*100;
+		hh15_35pct = ((hh2035 - hh2015)/hh2015)*100;
+		hh15_40pct = ((hh2040 - hh2015)/hh2015)*100;
+		hh15_45pct = ((hh2045 - hh2015)/hh2015)*100;
+
+	/*Percent change in population for base year 2010*/
+		tpop10_15pct = ((tpop2015 - tpop2010)/tpop2010)*100;
+		tpop10_20pct = ((tpop2020 - tpop2010)/tpop2010)*100;
+		tpop10_25pct = ((tpop2025 - tpop2010)/tpop2010)*100;
+		tpop10_30pct = ((tpop2030 - tpop2010)/tpop2010)*100;
+		tpop10_35pct = ((tpop2035 - tpop2010)/tpop2010)*100;
+		tpop10_40pct = ((tpop2040 - tpop2010)/tpop2010)*100;
+		tpop10_45pct = ((tpop2045 - tpop2010)/tpop2010)*100;
+
+	/*Percent change in household for base year 2015*/
+		tpop15_20pct = ((tpop2020 - tpop2015)/tpop2015)*100;
+		tpop15_25pct = ((tpop2025 - tpop2015)/tpop2015)*100;
+		tpop15_30pct = ((tpop2030 - tpop2015)/tpop2015)*100;
+		tpop15_35pct = ((tpop2035 - tpop2015)/tpop2015)*100;
+		tpop15_40pct = ((tpop2040 - tpop2015)/tpop2015)*100;
+		tpop15_45pct = ((tpop2045 - tpop2015)/tpop2015)*100;
+
+run;
+
+data bpk_popforecasts_trmerge;
+	merge bpk_popforecasts general.geo2010 (keep=tract tract6 ntract);
+run;	
+
+
+data bpk_popforecasts_tr10;
+	set bpk_popforecasts_trmerge;
+	where geo2010 = "11001006500" or geo2010 = "11001006600" or geo2010 = "11001006700" or
```

Comment:
This **where** statement is perfectly good. Just FYI, there's a slightly more concise way of writing this using an **in** operator. 

`where geo2010 in ( "11001006500", "11001006600", "11001006700", {fill in rest}, "11001007605" )`

But, what you have now is fine.